proto/ha_scope.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package dash.ha_scope;
+
+import "types.proto";
+
+message HaScope {
+    // HA scope ID. It can be the HA set id (scope = dpu) or ENI id (scope = eni)
+    string ha_scope_id = 1;
+    // Config version.
+    string version = 2;
+    // If true, disable this vDPU. It can only be false or true.
+    bool disabled = 3;
+    // The HA role for this scope.
+    types.HaRole ha_role = 4;
+    // If true, flow reconcile will be initiated. (Message Only. Not saved in DB.)
+    bool flow_reconcile_requested = 5;
+    // If true, HA role will be activated. (Message Only. Not saved in DB.)
+    bool activate_role_requested = 6;
+}
+
+message HaScopeKey {
+    // HA scope ID. It can be the HA set id (scope = dpu) or ENI id (scope = eni).
+    string ha_scope = 1;
```

Comment:
Is the expected value here supposed to be an ENI/HA set ID, or the literal strings "dpu" or "eni"?

Reply:
addressed